subsys/net/l2/openthread/Kconfig

```diff
@@ -330,5 +330,86 @@ config OPENTHREAD_REFERENCE_DEVICE
 	help
 	  Enable Thread Certification reference device support in OpenThread stack
 
+# Thread 1.2 not available
```

Comment:
Is there a point adding the commented out options in the first place? I think this could confuse users.

Reply:
The idea was to add them commented out to indicate there is a reason for not being available.
I can remove them if you want.